### DISCUSSION TITLE
Tanix Tx3 ; Aml multiboot not applying

### DISCUSSION THREAD

**Discussion by danidanibaerboo** [devmfc/debian-on-amlogic#73] [Q&A] [answered]
Hey everyone, I've been experimenting with Debian on a USB and everything has been working great. Both the Wi-Fi and Ethernet are functioning properly.

The issue I'm facing is that running  ./aml-multiboot.sh script, it doesn't return any errors. However, my TX3 won't boot from the USB automatically. I still have to hold down the reset button for it to boot from the USB.

Unfortunately, I only realized this after running ./install-to-emmc.sh , which resulted in a bootloop.

No other image/armbian/llinux has been flashed before.
Slimbox firmware installed prior to attempt.
Screenshots were from second attempts (after bootloop as I thought I did something wrong).

![IMG_2024-07-13-19-19-05-370](https://github.com/user-attachments/assets/e4a3a0fe-49a2-4882-b2a4-95e59a4f458b)
![IMG_2024-07-13-19-31-50-992](https://github.com/user-attachments/assets/6a355cd4-c454-4968-946e-5259d99c99f2)

**Comment by devmfc** [comment, marked as answer]
can you try:
replace /root/aml-multiboot-setup.sh with attached version and then run:

~~~
fw_setenv cmd_boot_org
./aml-multiboot-setup.sh
fw_printenv
~~~

and post the output here?
[aml-multiboot-setup.zip](https://github.com/user-attachments/files/16224641/aml-multiboot-setup.zip)


My images are created for stock vendor bootloaders. I don't know what version bootloader SlimBox uses. But if it doesn't work, you will have to go back to stock firmware before installing multiboot.

**Reply by danidanibaerboo** [reply]
I discovered that I have a fake uboot, 

which  bricked my device when I tried to multiboot and install it to the emmc (weird as it didn't throw any error). 
Tried  AML burn tool,  it got stuck at 7% with a uboot error (QZ and BZ firmware).

Ended up returning the device and got a new one . Retried all previous step and got it working perfectly.

Thanks for your work!
